model/aspnetcore/metrics.yaml

```diff
@@ -114,3 +125,166 @@ groups:
+    brief: Duration of processing component parameters.
+    instrument: histogram
+    unit: "s"
+    note: |
+      Meter name: `Microsoft.AspNetCore.Components.Lifecycle`; Added in: ASP.NET Core 10.0
+    extends: aspnetcore.components.lifecycle.metrics.attributes
+    attributes:
+      - ref: aspnetcore.components.type
+        requirement_level: required
+
+  - id: metric.aspnetcore.components.update_parameters.exceptions
+    type: metric
+    metric_name: aspnetcore.components.update_parameters.exceptions
+    stability: development
+    brief: Total number of exceptions during processing component parameters.
+    instrument: counter
+    unit: "{exception}"
+    note: |
+      Meter name: `Microsoft.AspNetCore.Components.Lifecycle`; Added in: ASP.NET Core 10.0
+    extends: aspnetcore.components.lifecycle.metrics.attributes
+    attributes:
+      - ref: aspnetcore.components.type
+        requirement_level: required
+      - ref: error.type
+        brief: The full name of exception type.
+        examples: ['System.OperationCanceledException', 'Contoso.MyException']
+        requirement_level: required
+
+  - id: metric.aspnetcore.components.rendering.batch.duration
+    type: metric
+    metric_name: aspnetcore.components.rendering.batch.duration
+    stability: development
+    brief: Duration of rendering batch.
+    instrument: histogram
+    unit: "s"
+    note: |
+      Meter name: `Microsoft.AspNetCore.Components.Lifecycle`; Added in: ASP.NET Core 10.0
+    extends: aspnetcore.components.lifecycle.metrics.attributes
+    attributes:
+      - ref: aspnetcore.components.diff.approximate.length
+        requirement_level: required
+
+  - id: metric.aspnetcore.components.rendering.batch.exceptions
+    type: metric
+    metric_name: aspnetcore.components.rendering.batch.exceptions
+    stability: development
+    brief: Total number of exceptions during batch rendering.
+    instrument: counter
+    unit: "{exception}"
+    note: |
+      Meter name: `Microsoft.AspNetCore.Components.Lifecycle`; Added in: ASP.NET Core 10.0
+    extends: aspnetcore.components.lifecycle.metrics.attributes
+    attributes:
+      - ref: error.type
+        brief: The full name of exception type.
+        examples: ['System.OperationCanceledException', 'Contoso.MyException']
+        requirement_level: required
+
+  # Blazor circuits
+  - id: metric.aspnetcore.components.circuits.count
+    type: metric
+    metric_name: aspnetcore.components.circuits.count
+    stability: development
+    brief: Total number of circuits.
+    instrument: counter
+    unit: "{circuit}"
+    note: |
+      Meter name: `Microsoft.AspNetCore.Components.Server.Circuits`; Added in: ASP.NET Core 10.0
+    extends: aspnetcore.components.circuits.metrics.attributes
+
+  - id: metric.aspnetcore.components.circuits.active_circuits
+    type: metric
+    metric_name: aspnetcore.components.circuits.active_circuits
+    stability: development
+    brief: Number of active circuits.
```

Comment:
what's the difference between active and connected circuits below? Can you add a note to explain?

Reply:
done